tools/generate-maximal-storage-format-rdg/generate-maximal-storage-format-rdg.cpp

```diff
@@ -76,7 +91,10 @@ main(int argc, char** argv) {
   cll::ParseCommandLineOptions(argc, argv);
 
   KATANA_LOG_ASSERT(!InputFile.empty());
-  MaximizeGraph(InputFile, OutputFile);
+  auto res = MaximizeGraph(InputFile, OutputFile);
+  if (!res) {
+    KATANA_LOG_FATAL("failed to generate maximal graph");
```

Comment:
What is this?

Reply:
the maximal rdg is how we attempt to catch unintended changes to the rdg storage format. Its not a perfect test, but was quick & easy to implement so it will do for now. 
The idea is:
1) we generate an RDG with all optional items present at the current storage_format_version. This becomes our known good maximal RDG.
2) CI runs a test which load/store cycles the known good maximal RDG, then compares the on disk structure of the original known good maximal RDG with the one written by the katana software under test.

Reply:
Thanks for the explanation!